lib/src/utils/utils.dart

```diff
@@ -15,3 +15,24 @@ bool ifContainsKeyAndNotEmpty<T>(Map<String, T> map, String key) {
 bool isStringEmpty(String? value) => !checkIfNotEmpty(value);
 
 bool isListEmpty(List? value) => !checkIfListIsNotEmpty(value);
+
+/// Converts passed double in K, L and Cr notion, significantly shrinking the length of given number.
+/// For example: A double 34394941.23 is not easily readable, now it will returns this value as 3.44Cr
+String getShrinkedLargeDouble(double value, {bool isReturnTypeDouble = false}) {
```

Comment:
the name should be formatPrice, or formatValue

Reply:
@Sam1kumar I also agree with Ayush here.

Reply:
Please also add test case for the same.